app/serializers/component_serializer/header_component_serializer.rb

```diff
@@ -3,6 +3,9 @@ class HeaderComponentSerializer < ComponentSerializer::BaseComponentSerializer
     # Initialise a header component.
     #
     # @param [Array<Object>] components items that form the children of the header component.
+    #
+    # @example Initialising a header component
+    #   ComponentSerializer::HeaderComponentSerializer.new([A_serializer, Another_serializer]).to_h
```

Comment:
This would be great if you could show an ample of initialising a something into the new call

Reply:
I am not sure what you mean. Do you mean you would like to see an actually serializer in there?

Reply:
Kind of, yeah - As a rule examples should be stand allows and executable. They can be on multiple lines so you could do something like:

```ruby
# @example
#   a_serialiser = ComponentSerialiser::ParagraphComponentSerialiser.new(['foo'])
#   another_serialiser =  ComponentSerialiser::ParagraphComponentSerialiser.new(['bar'])
#   ComponentSerializer::HeaderComponentSerializer.new([a_serialiser, another_serialiser]).to_h
```